src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,261 @@
+        boolean noTask = inputWords[0].trim().equals("event");
+        boolean noDate = inputWords.length == 1;
+        if (noTask || noDate) {
+            throw new DukeException("Your task is not in the right format");
+        }
+        description = inputWords[0].trim();
+        at = inputWords[1].trim();
+        System.out.println("Got it. I've added this task:");
+        tasks.add(new Event(description, at));
+        System.out.println(tasks.get(tasks.size() - 1).getTaskInfo());
+        System.out.println("Now you have " + tasks.size() + " tasks in the list");
+    }
+
+    private static void addDeadline(String input) throws DukeException {
+        String by;
+        String description;
+        String[] inputSplit = input.trim().split(" ");
+        boolean noInput = inputSplit.length == 1;
+        if (noInput) {
+            throw new DukeException("You have to specify the task!");
+        }
+        boolean noSeparator = !input.contains("/");
+        if (noSeparator) {
+            throw new DukeException("Your task is not in the right format");
+        }
+        String[] inputWords = input.split("/");
+        boolean noTask = inputWords[0].trim().equals("deadline");
+        boolean noDate = inputWords.length == 1;
+        if (noTask || noDate) {
+            throw new DukeException("Your task is not in the right format");
+        }
+        description = inputWords[0].trim();
+        by = inputWords[1].trim();
+        System.out.println("Got it. I've added this task:");
+        tasks.add(new Deadline(description, by));
+        System.out.println(tasks.get(tasks.size() - 1).getTaskInfo());
+        System.out.println("Now you have " + tasks.size() + " tasks in the list");
+    }
+
+    private static void addTodo(String input) throws DukeException {
+        String[] inputSplit = input.split(" ");
+        boolean noInput = inputSplit.length == 1;
+        if (noInput) {
+            throw new DukeException("You have to specify the task!");
+        }
+        System.out.println("Got it. I've added this task:");
+        tasks.add(new Todo(input));
+        System.out.println(tasks.get(tasks.size() - 1).getTaskInfo());
+        System.out.println("Now you have " + tasks.size() + " tasks in the list");
+    }
+
+    private static void listTasks() throws DukeException {
+        if (tasks.size() == 0) {
+            throw new DukeException("You do not have any tasks in your list!");
+        }
+        for (int i = 0; i < tasks.size(); i++) {
+            tasks.get(i).printTask(i + 1);
+        }
+    }
+
+    private static void printHelloMessage() {
+        System.out.println("Hello from\n" + LOGO);
+        System.out.println(LINE_SEPARATOR);
+        System.out.println(HELLO_MESSAGE);
+        System.out.println(LINE_SEPARATOR);
+    }
+
+    private static void printByeMessage() {
+        System.out.println(BYE_MESSAGE);
+        System.out.println(LINE_SEPARATOR);
+    }
+}
```

Comment:
You have packed most of the methods inside this one class. Please consider putting some methods in other classes. For example, all the task-related methods can be under a new class called TaskManager.